js/okx.js

```diff
@@ -2796,6 +2797,15 @@ module.exports = class okx extends Exchange {
                 }
                 request['ordType'] = ordType;
             }
+        } else {
```

Comment:
@sc0Vu  can you check this syntax error? 
![image](https://user-images.githubusercontent.com/43336371/200594197-f0ba6826-6c0a-4096-96b5-f57ab98fc11b.png)

Reply:
@carlosmiei updated, thanks!